gem/Rakefile

```
@@ -90,7 +91,12 @@ task :python_copy do
   copy_files(platform_config[:python], meterpreter_dest)
 end
 
-task :win_prep => [:create_dir, :win_compile, :win_copy] do
+task :bill_of_materials do
```

If we're going down the nomenclature of manifest:

```suggestion
task :create_manifest do
```